bench-vortex/src/engines/ddb/mod.rs

```diff
@@ -12,8 +14,8 @@ use crate::Format;
 use crate::datasets::BenchmarkDataset;
 
 /// Finds the path to the DuckDB executable
-pub fn executable_path(user_supplied_path_flag: &Option<path::PathBuf>) -> path::PathBuf {
-    let validate_path = |duckdb_path: &path::PathBuf| {
+pub fn build_and_get_executable_path(user_supplied_path_flag: &Option<PathBuf>) -> PathBuf {
```

Comment:
If we want to use a pre-installed binary, these steps need to be separate `build_and_get_executable_path`.

Reply:
Would still prefer though to keep these separate.

Reply:
Will do this in the next PR